style.css

```diff
@@ -1,13 +1,16 @@
+/* MAIN SECTION */
```

Comment:
Overall comment on the css. Look up css grid or flex box. I think you'll like it a lot more and save yourself a lot of code.

Reply:
Also try and make generic css class for elements that can be used for more than one block.

Reply:
This looks like I may have to redo the whole css, which I am down with haha but I would wanna make a new branch for it because it's something I wanna play around with more than just a quick transition

Reply:
Fair